app/models/invite_request.rb

```diff
@@ -30,15 +31,24 @@ def position
     InviteRequest.where("id <= ?", id).count
   end
 
-  #Ensure that invite request is for a new user
+  # Ensure that email is not banned
+  def check_admin_banned_list
+    return unless AdminBlacklistedEmail.exists?(email: self.email)
+
+    errors.add(:email, "has been blocked at the owner's request. That means it can't be used for invitations. 
```

Comment:
Could you i18n this by using a symbol and putting the string into the locale file? There's some docs here: https://github.com/otwcode/otwarchive/wiki/Internationalization-%28i18n%29-Standards#user-content-errorsadd